.travis.yml

```diff
@@ -136,8 +136,9 @@ env:
     - PROJECT_DIR=tests/issue/22 TOOLCHAIN=default
     - PROJECT_DIR=tests/issue/24/unit TOOLCHAIN=default
     - PROJECT_DIR=tests/issue/107 TOOLCHAIN=default
+    - PROJECT_DIR=tests/issue/109/unit TOOLCHAIN=default
     - PROJECT_DIR=tests/hunter_create_args_file TOOLCHAIN=default
     ### -- end
-
+    ###
```

Comment:
I guess it's not intended to be here :) If you're adding new commits to branch they will be added to pull request automatically. In future you can create separate branch for each pull request to avoid this.